src/CommonITILTask.php

```diff
@@ -291,10 +291,55 @@ private function handleTaskDuration(array &$input, int $timestart, int $timeend)
         }
     }
 
+    public function assignTechFromtask($input)
+    {
+        //if user or group assigned to ticket task, add it to the ticket
+        $entities = [
```

Comment:
"entities" is a bit ambiguous here because we often use it to refer to the entity feature of GLPI.
Maybe something like "itemtype" would be clearer ?